### PR TITLE
docs: update "move all data" instructions in FAQ

### PR DESCRIPTION
**Pull request by aberndsen**
chore(docs): update "move all data" FAQ instructions.

### CODE REVIEW

**Review by bo0tzz — APPROVED** [review]

File: docs/docs/FAQ.mdx [hunk 1]
```diff
@@ -140,6 +140,9 @@ UPDATE person SET "ownerId" = '<destinationId>' WHERE "ownerId" = '<sourceId>';
 // reassign assets
 UPDATE assets SET "ownerId" = '<destinationId>' WHERE "ownerId" = '<sourceId>'
  AND CHECKSUM NOT IN (SELECT CHECKSUM FROM assets WHERE "ownerId" = '<destinationId>');
+
+// (optionally) reassign external libraries
```
Comment (mmomjian) [inline docs/docs/FAQ.mdx:144]:
```suggestion
-- (optionally) reassign external libraries
```

File: docs/docs/FAQ.mdx [hunk 2]
```diff
@@ -140,6 +140,9 @@ UPDATE person SET "ownerId" = '<destinationId>' WHERE "ownerId" = '<sourceId>';
 // reassign assets
```
Comment (mmomjian) [inline docs/docs/FAQ.mdx:140]:
```suggestion
-- reassign assets
```

**Review by mmomjian — CHANGES_REQUESTED** [review]
Thank you for your PR. Could we change to SQL-style comments? We would need to also edit the "reassign albums" and "reassign people" blocks, which are just above the lines changed in this PR

**Comment by aberndsen** [comment]
Hi @mmomjian 

I've adjusted the "move all data" section to have sql-style comments throughout. I also "promoted" the optional changes to the **libraries** table, indicating there are four tables in total that should be changed if one wants to move all data to a different user.

**Pull request merged**